modules/identity-provider-configuring-apache-request-header.adoc

```diff
@@ -240,5 +240,7 @@ Enter your Kerberos user name and password at the prompt.
 # oc login
```

Comment:
@stlaz Can you help me with what this `oc login` command should be updated to? This PR is updating existing "oc login" commands to pass in a username, since per https://bugzilla.redhat.com/show_bug.cgi?id=1920427, "oc login" alone won't prompt for the username anymore.

I updated the first instance of this file's "oc login" to pass in the username. But I wasn't positive what behavior was expected in this step. Is it correct that they'll still have to provide the user name?

Existing section is here: https://docs.openshift.com/container-platform/4.7/authentication/identity_providers/configuring-request-header-identity-provider.html#identity-provider-configuring-apache-request-header_configuring-request-header-identity-provider

Reply:
You can keep the `oc login` here, the code changes to `oc login` should only apply to basic authentication, which is not the case here

Reply:
Excellent thank you!